### PR TITLE
[Hotfix]: Sync implementation With Design

### PR DESCRIPTION
**Pull request by erfan-goodarzi**
This PR syncs implementation with design.
## Changes

- remove `ssnInput`.
- reorder the create family button.

### CODE REVIEW

**Review by AmirabbasJ — APPROVED** [review]

File: app/components/HouseholderForm/HouseholderIdentityForm.tsx [hunk 1]
```diff
@@ -117,13 +113,20 @@ export const HouseholderIdentityForm = () => {
...
-            error={formState.errors.ssn?.message}
-            className={classes.textInput}
-            {...register('ssn')}
-            placeholder={t.ssnInput.placeholder}
-            label={`${t.ssnInput.label}:`}
+          <Select
+            wrapperProps={createTestAttr(Ids.genderInput)}
+            data={[
+              {
+                value: male.value,
```
Comment (ASafaeirad) [inline app/components/HouseholderForm/HouseholderIdentityForm.tsx:120]: The value is a domain concern, we shouldn't keep it in our messages.

**Review by ASafaeirad — CHANGES_REQUESTED** [review]

File: app/components/HouseholderForm/HouseholderIdentityForm.tsx (hunk 1 above)
Comment (erfan-goodarzi) [reply app/components/HouseholderForm/HouseholderIdentityForm.tsx:120]: So what about other select inputs?

Comment (ASafaeirad) [reply app/components/HouseholderForm/HouseholderIdentityForm.tsx:120]: It's true for all select.
We should change it everywhere

**Comment by erfan-goodarzi** [comment]
@ASafaeirad I moved the values to the domain but I'm not sure about that. is this the correct way of doing that?

**Comment by ASafaeirad** [comment]
No bro it's not good idea.

**Review by ASafaeirad — APPROVED** [review]

**Pull request merged**